bridge/bridge.go

```diff
@@ -0,0 +1,14 @@
+package bridge
+
+import (
+	"github.com/giantswarm/flanneltpr/docker"
+)
+
+// Bridge holds information used for the execution of the Docker entrypoint of
+// https://github.com/giantswarm/k8s-network-bridge.
+type Bridge struct {
+	// Config contains network bridge configuration.
+	Config Config `json:"config" yaml:"config"`
+	// Docker describes the docker image running k8s-network-bridge.
+	Docker docker.Docker `json:"docker" yaml:"docker"`
```

Comment:
My thinking was: this is a docker configuration. This isn't really `Bridge` or `Flannel` specific, so why repeat the code and create artificial directory structure with copied files. I find it more natural, and like it more. Shout at me if you want it changed, anyway.